src/components/Auth/Auth.tsx

```diff
@@ -105,10 +126,46 @@ function Auth() {
             <Typography.Text>{signup ? 'Already have an account?' : "Don't have an account?"}</Typography.Text> <br />
             <Typography.Link onClick={handleModeSwitch}>
               {signup ? 'Log in' : 'Limited time First 87 $MAGIC✨ users Free sign up'}
-            </Typography.Link>
+            </Typography.Link>{' '}
+            <br /> <br />
+            <Typography.Link onClick={handlePasswordReset}>{signup ? '' : 'Forgot Password?'}</Typography.Link>
```

Comment:
It's more semantically correct to remove the node completely (instead of hiding text) or to set visibility to hidden / display to none.